assets/js/components/setup/CompatibilityChecks/checks.js

```diff
@@ -27,7 +27,6 @@ import {
 	ERROR_GOOGLE_API_CONNECTION_FAIL,
 	ERROR_INVALID_HOSTNAME,
 	ERROR_TOKEN_MISMATCH,
-	ERROR_WP_PRE_V5,
```

Comment:
I noticed this is also being used here: https://github.com/google/site-kit-wp/blob/f63c6a6aabf829d797f11a1cd1e9ad313566887d/assets/js/components/setup/CompatibilityChecks/CompatibilityErrorNotice.js#L206-L214

Given Site Kit can't be installed on versions before WP 5.2 now, we might as well remove this constant/error reason from the entire codebase—this issue makes sense as the place to do so in so let's do it here 😄

Reply:
I searched and removed from a number of files.